calico_containers/pycalico/datastore_datatypes.py

```diff
@@ -42,7 +42,10 @@ def to_json(self):
         json_dict["inbound_rules"] = [rule.to_json_dict() for rule in rules]
         rules = json_dict["outbound_rules"]
         json_dict["outbound_rules"] = [rule.to_json_dict() for rule in rules]
-        return json.dumps(json_dict)
+        if pprint:
```

Comment:
or even pass through kwargs (stating that the kwargs are used for json.dumps formatting).

In either case, it seems better to have calicoctl decide on the formatting rather than it decided by the datastore.